main.go

```diff
@@ -339,19 +334,32 @@ To 'escape' those operators, please use single quotes, e.g.: 'uid>0'
 		}
 
 		if strings.HasPrefix("container", f) || (strings.HasPrefix("!container", f) && len(f) > 1) {
+			filter.NewPidFilter.Enabled = true
+			filter.NewPidFilter.Value = true
 			err := parseBoolFilter(f, filter.ContFilter)
 			if err != nil {
 				return tracee.Filter{}, err
 			}
 			continue
 		}
 
-		if filterName == "newpidns" || filterName == "!newpidns" {
-			err := parseBoolFilter(f, filter.NewPidNsFilter)
-			if err != nil {
-				return tracee.Filter{}, err
+		if strings.HasPrefix("container", filterName) {
```

Comment:
correct me if I'm wrong but I think that if `--trace container=new`, it will never reach this `if` because it will be caught in the previous `if strings.HasPrefix("container",f)`?

Reply:
Nope. the previous if means that "f" should be a prefix of "container".
If "f" equals "container=new", it is not a prefix of "container"